### ISSUE TITLE
[Accessibility][ScreenReader]: Links should be announced as links

### ISSUE THREAD

**Issue by rp-dev** [microsoft/BotFramework-WebChat#3115]
## Screenshot
![image](https://user-images.githubusercontent.com/50723361/79808648-4b618e00-8323-11ea-9315-eef08fbfdd88.png)

## Version
v4

## Describe the bug
The role "link" is not announced. 

## Steps to reproduce

JAWS reads " Terms and Conditions"  (without link).  

## Expected behavior
JAWS should read "Terms and Conditions link".   
Note:  Same is noticed for  "Privacy Statement"  link.   

[Bug]

**Comment by UncleZen** [comment]
Any updates on this issue? Will there be a fix any time soon?

**Comment by compulim** [comment]
This is fixed in #3287.
